pkgs/development/interpreters/bats/default.nix

```diff
@@ -2,20 +2,15 @@
 
 stdenv.mkDerivation rec {
   pname = "bats";
-  version = "1.2.0";
+  version = "1.2.1";
 
   src = fetchzip {
     url = "https://github.com/bats-core/bats-core/archive/v${version}.tar.gz";
-    sha256 = "0f59zh4d4pa1a7ybs5zl6h0csbqqv11lbnq0jl1dgwm1s6p49bsq";
+    hash = "sha256-grB/rJaDU0fuw4Hm3/9nI2px8KZnSWqRjTJPd7Mmb7s=";
   };
 
   patchPhase = ''
     patchShebangs ./install.sh
-
-    substituteInPlace ./libexec/bats-core/bats \
-        --replace 'type -p greadlink readlink' 'type -p ${coreutils}/bin/readlink'
-    substituteInPlace ./libexec/bats-core/bats-format-tap-stream \
-        --replace grep ${gnugrep}/bin/grep
```

Comment:
I'm less sure about this one.

It looks like `bats-format-tap-stream` was renamed `bats-format-pretty` in https://github.com/bats-core/bats-core/pull/246 (see: https://github.com/bats-core/bats-core/commit/3d7e49f642ab34b22c0068c43c8e671d0e03c84e#diff-870c4614c5e0940e0d7208f7d67375d3927f50ea8b87d2ef6d2638593105f78cR314), and I don't see any grep references in it.